### ISSUE TITLE
Reserve part of hash for future use

### ISSUE THREAD

**Issue by whoabuddy** [bitcoin-nfts/bitcoin-colors#37]
After thinking about increasing the size of the resulting hashed string in #34, it might be worth reserving some unique characters for future use.

Consider the sha512 output below:

```
Address: bc1q962j6tf9hyxeuhj9e3u0zhgn6dqjcrlz9cljmr
Address hash: 3dc5ad652dccbc986add9b41922d578478bca4d4df799ec5f5d70267a9efed64c1c7bc88c8a1ce7e6ee608095bb9dd57651093d1650f78c081af59c6f30ead1f
Substring 0: 3dc5ad652dccbc98 - 0.24129756660640828
...
Substring 7: 81af59c6f30ead1f - 0.5065818892208493
```

The substrings for 8 trait layers are 16 characters in length. What if we created 2 or more "empty" trait layers that could be filled in later? This could be used for future trait layers (think surprises, evolutions, expansions) or for the client-side trait ideas as well.

**Comment by whoabuddy** [comment]
Input from 2022/01/05:

- could change shape of JSON with top-level metadata
- specify number of reserved bits
- trait definitions in another attribute

**Comment by whoabuddy** [comment]
Fixed in #41
